ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/versions/electra/block/BlockProcessorElectra.java

```diff
@@ -360,36 +390,6 @@ public void processWithdrawalRequests(
         });
   }
 
-  /*
-   Implements process_deposit_request from consensus-specs (EIP-6110)
-  */
```

Comment:
moving all of these fns does make it a lot harder to determine if there were changes in githubs ui...